hopsworks-rest-utils/src/main/java/io/hops/hopsworks/restutils/RESTCodes.java

```diff
@@ -1414,7 +1414,7 @@ public String toString() {
     TRAININGDATASETJOB_FAILURE(80, "Could not write featurestore cloud args to HDFS",
       Response.Status.INTERNAL_SERVER_ERROR),
     TRAININGDATASETJOB_DUPLICATE_FEATURE(81, "Feature list contains duplicate", Response.Status.BAD_REQUEST),
-    TRAININGDATASETJOB_FEATURE_NOT_EXISTING(82, "Feature does not exist", Response.Status.BAD_REQUEST),
+    FEATURE_NOT_EXISTING(82, "Feature does not exist", Response.Status.BAD_REQUEST),
```

Comment:
@SirOibaf since we are changing the name, we can improve it to `FEATURE_DOES_NOT_EXIST`

Reply:
Fixed.